crates/ruff_linter/src/rules/isort/settings.rs

```diff
@@ -75,6 +75,29 @@ pub struct Settings {
     pub length_sort_straight: bool,
 }
 
+impl Settings {
+    pub fn requires_module_import(&self, name: String, as_name: Option<String>) -> bool {
```

Comment:
In the `isort` implementation we are doing the reverse thing: iterating through the required imports and unpacking them to see if they are found (and adding them if not), rather than packaging up some strings into a named import and then checking if the isort settings contain that. So I'm not sure how to use these specific methods in that case. (apologies again if I'm misunderstanding something here!)